tests/boot/uefi_bootmenu.pm

```diff
@@ -1,43 +0,0 @@
-# SUSE's openQA tests
-#
-# Copyright © 2016 SUSE LLC
-#
-# Copying and distribution of this file, with or without modification,
-# are permitted in any medium without royalty provided the copyright
-# notice and this notice are preserved.  This file is offered as-is,
-# without any warranty.
-
-# Summary: Select UEFI boot device in BIOS
-#    OVMF doesn't honor the -boot XX setting of qemu so we have to manually enter the boot manager in the BIOS
-# Maintainer: Ludwig Nussel <ludwig.nussel@suse.de>
-
-use base "basetest";
-use strict;
-use testapi;
-use utils;
-use bootloader_setup;
-
-sub run {
-    my ($self) = @_;
-    # because of broken firmware, bootindex doesn't work on aarch64 bsc#1022064
-    # selecting a workaround is handled in boot/boot_to_desktop
-    return if (get_var('MACHINE') =~ qr'aarch64' && get_var('BOOT_HDD_IMAGE'));
```

Comment:
New week and a coffee! After looking more at the code (and after @okurz remark), this code (uefI_bootmenu.pm) is only executed on UEFI and arch other than aarch64 (so mainly x86_64) with`BOOT_HDD_IMAGE` set. So sounds better to keep it.

But I don't understand why is it executed on aarch64 if `BOOTFROM` is set? But I don't have enough time to analyze this, I would like to have this merged for RC3 BV.

So I will re-add it with just the arch check modified.